### DISCUSSION TITLE
is there a way to add multiple folders to a playlist?

### DISCUSSION THREAD

**Discussion by danielb2** [namidaco/namida#80] [General]
Seems I could only click one at a time to get them added

**Comment by danielb2** [comment]
also could we get a confirmation when adding music to a playlist that already are int he playlist?

**Comment by MSOB7YY** [comment]
yeah folder selection is not there
idk if i can make this possibly currently, it will require some refactor

about confirmation, it will make it slower while adding but sure gonna look into this

**Reply by danielb2** [reply]
when I click the hamburger menu, there is an option to add to playlist. but it only lets me select individual tracks as opposed to folders. i think that's an obvious easy to change

**Reply by MSOB7YY** [reply]
uhmmm actually not at all
the selection itself is not hard, but the moment u start working on it, u gonna find urself taking care of:
- code: REFACTOR to allow inserting folders in the same list
- or if separate lists, u gonna force user to only select folders
- ui: either u chose first or second, there will be a huge ui adaptation, separate functions to handle both cases, more more.. its pain really

im not saying it's impossible but it will surely take time, which i want to invest for YouTube/video playback currently

**Reply by danielb2** [reply]
sorry, I didn't mean th coding is easy to change. My meaning is that it's an obvious place to add the change to

you could have one list for tracks and one list for folders?  Like you do for selecting folders and folders to exclude
